### PR TITLE
add express-wechat-access types

### PR DESCRIPTION
**Pull request by bdwenxi**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [x] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Review by plantain-00 — CHANGES_REQUESTED** [review]
The package is written by typescript, it contains types: https://github.com/simmons8616/express-wechat-access/blob/master/package.json#L5

**Comment by simmons8616** [comment]
@plantain-00 
I have deleted the types in package.json. （https://github.com/simmons8616/express-wechat-access/blob/master/package.json#L5）

File: types/express-wechat-access/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": false,
```
Comment (plantain-00) [inline types/express-wechat-access/tsconfig.json]: For new packages, it should enabled by default

File: types/express-wechat-access/tslint.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "extends": "dtslint/dt.json",
```
Comment (plantain-00) [inline types/express-wechat-access/tslint.json]: For new packages, this file should only `"extends": "dtslint/dt.json"`

**Review by plantain-00 — CHANGES_REQUESTED** [review]

**Comment by simmons8616** [comment]
@plantain-00 
Thank you for your suggestion, I have corrected all of them according to your suggestions.

**Review by plantain-00 — APPROVED** [review]

**Comment by DanielRosenwasser** [comment]
Thanks all!

**Pull request merged**